tools/wiktionary2lex/wiktionary2lex.py

```diff
@@ -30,9 +32,52 @@ def print_output(output):
                 print key + ":" + list_item[key] + "\t",
         print "\n"
 
+def output2lex(output,filename):
+    lexicon_output = open(filename,"wb")
+    lexicon = ET.Element("lexicon")
+    tree = ET.ElementTree(lexicon)
+
+    for list_item in output:
+        lex_entry = ET.SubElement(lexicon,"lex")
+        if list_item["pron"] is not None:
+            lex_entry.attrib["ipa"]=list_item["pron"]
+        if list_item["x-sampa"] is not None:
+            lex_entry.attrib["x-sampa"]=list_item["x-sampa"]
+        lex_entry.text=list_item["word"]
+
+    tree.write(
+            lexicon_output,pretty_print=True,   
+            xml_declaration=True,encoding="utf-8")
+
+def compare(dictionary1,dictionary2,*filters):
+    identical = 1
+    for key in filters:
+        if dictionary1[key] != dictionary2[key]:
+            identical = 0
+            break
+    return identical
+
+def filter_list(data,*filters):
```

Comment:
filter_list will be adding a new 'unique' attribute to each entry. To me a filtering process shouldn't change the entries, only remove some of the entries.